template/template.go

```diff
@@ -336,7 +339,13 @@ func (te Expander) Expand() (result string, resultErr error) {
 
 	templateTextExpansionTotal.Inc()
 
-	tmpl, err := text_template.New(te.name).Funcs(te.funcMap).Option("missingkey=zero").Parse(te.text)
+	tmpl := text_template.New(te.name).Funcs(te.funcMap)
+	if len(te.options) > 0 {
```

Comment:
This if should not be needed.

I'd suggest to set the options to []string{"missingkey=zero"} in NewTemplateExpander, if `te.options` is nil. Note that nil is different than > 0; you could pass an empty list if you do not want options. nil would mean give me the default option.